File: QuasarCode_Library/Games/Cards/IPlayingCard.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuasarCode.Library.Games.Cards
{
    /// <summary>
    /// A card from a standard deck of 52 playing cards
    /// </summary>
    public interface IPlayingCard
    {
        /// <summary>
        /// The card's value
        /// </summary>
        PlayingCard.AllowedValues Value { get; }

        /// <summary>
        /// The card's suit
        /// </summary>
        PlayingCard.AllowedSuits Suit { get; }
    }
}

File: QuasarCode_Python/QuasarCode/Games/Cards/_IPlayingCard.py
from enum import Enum

# Class to help prevent circular reference. See EOF for import
class PlayingCard(object):
    class AllowedValues(Enum):
        pass
    class AllowedSuits(Enum):
        pass

class IPlayingCard(object):
    """
    A card from a standard deck of 52 playing cards
    """

    @property
    def Value(self) -> PlayingCard.AllowedValues:
        """
        The card's value
        """
        raise NotImplementedError()

    @property
    def Suit(self) -> PlayingCard.AllowedSuits:
        """
        The card's suit
        """
        raise NotImplementedError()